Makefile

```diff
@@ -20,8 +20,11 @@ client:
 	$(PY) -m juju.client.facade -s "juju/client/schemas*" -o juju/client/
 
 .PHONY: test
-test:
-	tox
+test: lint
+	tox -e py3
+	@for f in $(shell find tests/integration -maxdepth 1 -mindepth 1 -name "*.py" | grep -v "__init__.py"); do \
+		tox -e integration -- "$${f}"; \
+	done
```

Comment:
I would rather see this done in the tox.ini file. There are some examples here: https://tox.readthedocs.io/en/latest/example/pytest.html

The point of tox is to make this sort of invocation unnecessary :-)